Baekjoon/2438.py

```diff
@@ -0,0 +1,10 @@
+'''
+문제 링크: https://www.acmicpc.net/problem/2438
+카테고리: 구현
+문제 해설: 별찍기
+'''
+n=int(input())
+
+for i in range(1,n+1):
+  print('*'*i)
+  i+=1
```

Comment:
```suggestion
  print('*'*i)
```

for 반복문에서 i+=1 은 동작하지 않습니다